bootcamp/javascript/arrays/array-advanced/solution/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Advanced arrays solution</title>
+</head>
+<body>
+<h1>Check the respective Exercise javascript files for solutions, as well as console for output</h1>
+
+<script src="exercise-1.js"></script>
```

Comment:
Since you can now add multiple exercises to a single topic I would suggest to split these exercises